mbuild/lattice.py

```diff
@@ -15,68 +14,66 @@ class Lattice(object):
     """Develop crystal structure from user defined inputs.
 
     Lattice, the abstract building block of a crystal cell.
-    Once defined by the user, the crystal is returned as
-    a single Compound that can be either replicated through its class
-    methods or through a similar replicate Compound method.
-
-    Lattice is defined through the standard bravais lattices, which have been
-    accepted by the International Union of Crystallography.
-    A Lattice can be fully described with its lattice vectors and lattice
-    spacings. Also, the Lattice can be fully defined by its lattice parameters:
-    the lattice spacings and its set of coordinate angles will then
-    generate the lattice vectors. Lattice expects a right handed lattice and
-    cell edges defined by vectors all originating from the origin in
-    Cartesian space.
+    Once defined by the user, the lattice can then be populated with
+    Compounds and replicated as many cell lengths desired in 3D space.
+
+    A Lattice is defined through the Bravais lattice definitions. With edge
+    vectors a1, a2, a3; lattice spacing a,b,c; and lattice points at unique
+    fractional positions between 0-1 in 3 dimensions. This encapsulates
+    distance, area, volume, depending on the parameters defined.
+
 
     Parameters
     ----------
-    dimension : int, optional, default=3
-        Dimension of the system of interest.
-    lattice_vectors : numpy array, shape=(dimension, dimension), optional
-                      default=([1,0,0], [0,1,0], [0,0,1])
-        Vectors that define edges of unit cell corresponding to dimension.
-    lattice_spacings : list-like, shape=(dimension,), optional, default=None
-        Length of unit cell edges.
-    basis_atoms : dictionary, shape={'id':[nested list of coordinate pairs]}
-                    default={'default':[[0., 0., 0.]]
-        Location of all basis Compounds in unit cell.
-    angles : list-like,  shape=(dimension,), optional, default=None
-        Interplanar angles describing unit cell.
+    lattice_spacing : numpy array, shape=(3,), required, dtype=float
+        Array of lattice spacings a,b,c for the cell.
+    lattice_vectors : numpy array, shape=(3, 3), optional
+                      default=[[1,0,0], [0,1,0], [0,0,1]]
+        Vectors that encase the unit cell corresponding to dimension. Will
+        only default to these values if no angles were defined as well.
+    lattice_points : dictionary, shape={'id': [[nested list of positions]]
+        optional, default={'default': [[0.,0.,0.]]}
+        Locations of all lattice points in cell using fractional coordinates.
+    angles : numpy array, shape=(3,), optional, dtype=float
+        Array of inter-planar Bravais angles
```

Comment:
Should specify the expected units. We've been using radians for other mBuild functionality (such as Compound.rotate, so it might be a good idea to use radians here as well for consistency.

Reply:
One argument against this would be due to the standardized crystallographic information formats from [IUCr](http://www.iucr.org/) which in their [standardized CIF file format](http://it.iucr.org/Ga/ch4o1v0001/Icell_angle_.html) their angles are in degrees.

Reply:
That makes sense. As long as the docstring specifies degrees then I think we're good.